app/webhook.go

```diff
@@ -122,7 +122,15 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 							webhookResp.Props = make(model.StringInterface)
 						}
 						webhookResp.Props["webhook_display_name"] = hook.DisplayName
-						if _, err := a.CreateWebhookPost(hook.CreatorId, channel, *webhookResp.Text, webhookResp.Username, webhookResp.IconURL, webhookResp.Props, webhookResp.Type, postRootId); err != nil {
+
+						text := a.ProcessSlackText(*webhookResp.Text)
```

Comment:
I noticed that the text in the webhook response isn't being processed as a Slack text, so I added this here. It parses tokens like <!here>, <!all>, <@userid>. If it is not relevant or needed, I can remove it.